src/components/Compare/Compare.style.tsx

```diff
@@ -240,6 +254,29 @@ export const ViewAllLink = styled.div`
   cursor: pointer;
 `;
 
+export const DisclaimerWrapper = styled.div`
+  background: #fafafa;
+  max-width: 345px;
+  width: 100%;
+  padding: 0.75rem;
+  line-height: 1.4;
+  margin: 1rem auto 0;
+
+  span {
+    color: rgba(0, 0, 0, 0.7);
+  }
+
+  a {
+    color: ${COLOR_MAP.BLUE};
+    text-decoration: none;
+  }
```

Comment:
The blue color should work automatically (it's a global style), but you can keep it if it doesn't

Reply:
hm yeah it was being overwritten by another style. gonna keep